client/src/components/EditProfile/EditMenu.tsx

```diff
@@ -130,7 +128,6 @@ const EditMenu = (): JSX.Element => {
         >
           <Tab label="Edit Profile" {...a11yProps(0)} />
           <Tab label="Profile Photo" {...a11yProps(1)} />
-          <Tab label="Availability" {...a11yProps(2)} />
```

Comment:
Can we move the availability tab here in order to remain consistent with the mocks?